library/sql_upgrade_fx.php

```diff
@@ -1068,6 +1068,8 @@ function upgradeFromSqlFile($filename, $path = '')
                 echo "<p class='text-success'>$skip_msg $line</p>\n";
             }
         } elseif (preg_match('/^#IfMappingUuidNeedUpdate/', $line)) {
+            echo "<p>Checking for missing resource UUIDs</p>";
```

Comment:
would throw in a mapped in here since the mapped uuids are different than the  non-mapped uuids:
```
Checking for missing mapped resource UUIDs
```

Reply:
as an aside, if have time and become re-obsessed with uuids :) , am thinking of migrating to just one sql upgrade function that does all of the uuids like the background service function does (now that we have mappings of the stuff inside the table).